### DISCUSSION TITLE
[Feature] Add Immich as an option in "Share to" on Android

### DISCUSSION THREAD

**Discussion by diablo581** [immich-app/immich#11203] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If Immich was an option for the "Share to" menu in android then it might be possible to easily export your photos from Google Photos to Immich using your Android phone as a bridge. Select a month, share to, Immich and let it pull them down and push them into the system.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2659
